cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -936,6 +937,101 @@ impl Inst {
             srcloc,
         }
     }
+
+    /// Choose which instruction to use for moving a value between registers.
+    pub(crate) fn r#move(
+        ty: Type,
+        from_reg: Reg,
+        to_reg: Writable<Reg>,
+        _srcloc: Option<SourceLoc>,
+    ) -> Inst {
```

Comment:
Looking around more, I guess this should just be merged in with `gen_move`?

Reply:
Ah, yes, that would actually be the best solution, I think.